dosbox-staging is failing to build on armv6

Looks like a similar or the same issue as before with libspeexdsp

```
git clone --recursive --depth 1 --shallow-submodules --branch v0.80.1 "https://github.com/dosbox-staging/dosbox-staging.git" "/home/pi/RetroPie-Setup/tmp/build/dosbox-staging"
```

```
[17/398] Compiling C object subprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p/resample.c.o
FAILED: subprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p/resample.c.o 
cc -Isubprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p -Isubprojects/speexdsp-1.2.1/libspeexdsp -I../subprojects/speexdsp-1.2.1/libspeexdsp -Isubprojects/speexdsp-1.2.1 -I../subprojects/speexdsp-1.2.1 -Isubprojects/speexdsp-1.2.1/include -I../subprojects/speexdsp-1.2.1/include -Isubprojects/speexdsp-1.2.1/include/speex -I../subprojects/speexdsp-1.2.1/include/speex -fvisibility=hidden -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -O3 -DHAVE_CONFIG_H -march=armv7-a -mfpu=neon -mcpu=arm1176jzf-s -mfpu=vfp -O2 -MD -MQ subprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p/resample.c.o -MF subprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p/resample.c.o.d -o subprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p/resample.c.o -c ../subprojects/speexdsp-1.2.1/libspeexdsp/resample.c
cc1: warning: switch -mcpu=arm1176jzf-s conflicts with -march=armv7-a switch
/tmp/ccpBz8x4.s: Assembler messages:
/tmp/ccpBz8x4.s:116: Error: selected processor does not support `vld1.32 {q4},[r1]!' in ARM mode
/tmp/ccpBz8x4.s:117: Error: selected processor does not support `vld1.32 {q8},[r3]!' in ARM mode
/tmp/ccpBz8x4.s:119: Error: selected FPU does not support instruction -- `vmul.f32 q0,q4,q8'
/tmp/ccpBz8x4.s:122: Error: selected processor does not support `vld1.32 {q4,q5},[r1]!' in ARM mode
/tmp/ccpBz8x4.s:123: Error: selected processor does not support `vld1.32 {q8,q9},[r3]!' in ARM mode
/tmp/ccpBz8x4.s:124: Error: selected processor does not support `vld1.32 {q6,q7},[r1]!' in ARM mode
/tmp/ccpBz8x4.s:125: Error: selected processor does not support `vld1.32 {q10,q11},[r3]!' in ARM mode
/tmp/ccpBz8x4.s:127: Error: selected FPU does not support instruction -- `vmul.f32 q0,q4,q8'
/tmp/ccpBz8x4.s:128: Error: selected FPU does not support instruction -- `vmul.f32 q1,q5,q9'
/tmp/ccpBz8x4.s:129: Error: selected FPU does not support instruction -- `vmul.f32 q2,q6,q10'
/tmp/ccpBz8x4.s:130: Error: selected FPU does not support instruction -- `vmul.f32 q3,q7,q11'
/tmp/ccpBz8x4.s:132: Error: selected processor does not support `vld1.32 {q4,q5},[r1]!' in ARM mode
/tmp/ccpBz8x4.s:133: Error: selected processor does not support `vld1.32 {q8,q9},[r3]!' in ARM mode
/tmp/ccpBz8x4.s:134: Error: selected processor does not support `vld1.32 {q6,q7},[r1]!' in ARM mode
/tmp/ccpBz8x4.s:135: Error: selected processor does not support `vld1.32 {q10,q11},[r3]!' in ARM mode
/tmp/ccpBz8x4.s:137: Error: selected FPU does not support instruction -- `vmla.f32 q0,q4,q8'
/tmp/ccpBz8x4.s:138: Error: selected FPU does not support instruction -- `vmla.f32 q1,q5,q9'
/tmp/ccpBz8x4.s:139: Error: selected FPU does not support instruction -- `vmla.f32 q2,q6,q10'
/tmp/ccpBz8x4.s:140: Error: selected FPU does not support instruction -- `vmla.f32 q3,q7,q11'
/tmp/ccpBz8x4.s:142: Error: selected FPU does not support instruction -- `vadd.f32 q4,q0,q1'
/tmp/ccpBz8x4.s:143: Error: selected FPU does not support instruction -- `vadd.f32 q5,q2,q3'
/tmp/ccpBz8x4.s:145: Error: selected FPU does not support instruction -- `vadd.f32 q0,q4,q5'
/tmp/ccpBz8x4.s:147: Error: selected processor does not support `vld1.32 {q6},[r1]!' in ARM mode
/tmp/ccpBz8x4.s:148: Error: selected processor does not support `vld1.32 {q10},[r3]!' in ARM mode
/tmp/ccpBz8x4.s:150: Error: selected FPU does not support instruction -- `vmla.f32 q0,q6,q10'
/tmp/ccpBz8x4.s:152: Error: selected FPU does not support instruction -- `vadd.f32 d0,d0,d1'
/tmp/ccpBz8x4.s:153: Error: selected processor does not support `vpadd.f32 d0,d0,d0' in ARM mode
/tmp/ccpBz8x4.s:1645: Error: selected FPU does not support instruction -- `vcvt.s32.f32 d0,d0,#15'
/tmp/ccpBz8x4.s:1646: Error: selected processor does not support `vqrshrn.s32 d0,q0,#15' in ARM mode
/tmp/ccpBz8x4.s:1647: Error: selected FPU does not support instruction -- `vmov.s16 r3,d0[0]'
[18/398] Compiling C object subprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p/preprocess.c.o
cc1: warning: switch -mcpu=arm1176jzf-s conflicts with -march=armv7-a switch
[19/398] Compiling C object subprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p/mdf.c.o
cc1: warning: switch -mcpu=arm1176jzf-s conflicts with -march=armv7-a switch
[20/398] Compiling C object subprojects/speexdsp-1.2.1/libspeexdsp/libspeexdsp.a.p/smallft.c.o
cc1: warning: switch -mcpu=arm1176jzf-s conflicts with -march=armv7-a switch
ninja: build stopped: subcommand failed.
```

```
speexdsp 1.2.1

  Library Summary
    Optimization level: 3
    SIMD instructions : NEON (armv7-a)
    Numerical type    : floating-point
    FFT library       : OggVorbis FFT (built-in)
```

--- a/scriptmodules/emulators/dosbox-staging.sh
+++ b/scriptmodules/emulators/dosbox-staging.sh
@@ -40,6 +40,9 @@ function build_dosbox-staging() {
     local meson_cmd="meson"
     [[ -f "$md_build/meson/meson.py" ]] && meson_cmd="python3 $md_build/meson/meson.py"
 
+    # disable speexdsp simd support on armv6 devices
+    isPlatform "arm6" && params+=(-Dspeexdsp:simd=false)
+
     $meson_cmd setup "${params[@]}" build
     $meson_cmd compile -j${__jobs} -C build
 

